l10n/en/products/vpn/seo/desktop.ftl

```diff
@@ -0,0 +1,48 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
+
+
+# Shared
+vpn-shared-cta = See pricing & availability
+vpn-shared-day-guarantee = 30-day money-back guarantee
+vpn-shared-what-youll-get = What you’ll get with { -brand-name-mozilla-vpn }:
+
+vpn-shared-feature-item-protection = Protection for up to 7 devices
+vpn-shared-feature-item-access = Access to servers in 30+ countries
+vpn-shared-feature-item-encryption = Device-level encryption
+vpn-shared-feature-item-bandwidth = No bandwidth restrictions
+vpn-shared-feature-item-logs = No online activity logs now or ever
```

Comment:
These will be removed/change pending PR #9979

Reply:
Updated to closer match PR #9979. We will need to revise this when that lands/copy is locked.